### ISSUE TITLE
Make http://box/maps & http://box/map work with whatever maps the implementer has...most recently installed? [OSM]

### ISSUE THREAD

**Issue by holta** [iiab/iiab#1496]
@tim-moody @m-anish @georgejhunt see #877 which is our umbrella ticket of OSM needs & design issues.

**Comment by holta** [comment]
http://box/maps should now work, thanks to PR #1724 

http://box/map doesn't not work, but that usage is not nearly as much requested, so let's let that go.

**Comment by holta** [comment]
@georgejhunt when adding a Map Pack using http://box.lan/admin > `Install Content` > `Add Map Region` the link from http://box/maps (to the latest installed) is not always created.

In fact the link reverted to http://box/osm-vector-maps/maplist/ in 3 different cases when I installed/upgraded a new Map Pack on these 3 OS's {Raspbian, Ubuntu 18.04, Ubuntu 16.04}.

I don't understand the situation (likely related to https://github.com/iiab/iiab/blob/master/roles/osm-vector-maps/templates/iiab-update-map#L53-L79 ?) but hopefully you can help me work it out and solve it!

**Comment by holta** [comment]
Same problem on Avni's machines, when their IIAB software was being updated.

Until http://box/maps is fixed in a more automatic way (hopefully before IIAB 7.0's release in July 2019) the manual workaround is to go into this file each time:
```
/library/www/osm-vector-maps/index.html
```
As it keeps getting reverted to the following, against the wishes of the implementer:
```
<head>
  <meta http-equiv="refresh" content="0; URL=/osm-vector-maps/maplist/" />
</head>
```
So the manual fix is to set it to something like the following:
```
<head>
  <meta http-equiv="refresh" content="0; URL=/osm-vector-maps/en-osm-omt_central_america_2017-07-03_v0.23/" />
</head>
```

**Comment by holta** [comment]
Fixed by PR #1834 -> PR #1835
